collector.go

```diff
@@ -167,6 +169,7 @@ func (s *SmokepingCollector) Collect(ch chan<- prometheus.Metric) {
 			stats.IPAddr.String(),
 			stats.Addr,
 			pinger.Source,
+			fmt.Sprintf("%d", pinger.TrafficClass()),
```

Comment:
Why not use `strconv` for this simple conversion?

```suggestion
			strconv.Itoa(pinger.TrafficClass()),
```

Reply:
hmmm, lapse of the mind I'm guessing. I'm wondering if there is a better use case here for maybe `strconv.FormatUint` since the `TrafficClass` is a `uint8`. However just warpping the `uint8` in the `int` and then using `strconv.Itoa` would be also a valid operation here.

Reply:
I've switched to `strconv.Itoa` where I used the sprint earlier.